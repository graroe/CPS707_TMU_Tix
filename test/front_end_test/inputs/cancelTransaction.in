login
admin
create
CirqueDuSoleil
20231027
!q
add
!q
delete
!q
logout