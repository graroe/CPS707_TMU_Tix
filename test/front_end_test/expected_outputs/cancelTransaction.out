Enter session type <sales or admin>: 
Admin login successful.
Enter new event name <max 15 chars>: 
Enter date of event <format YYYYMMDD>: 
Enter number of available tickets <max 9999>: 
Transaction canceled
Enter event name: 
Transaction canceled
Enter event name: 
Transaction canceled
Session terminated.